### PR TITLE
Feature/stork analytics

### PR DESCRIPTION
**Pull request by learykara**

### CODE REVIEW

File: src/analytics/actions.es6 [hunk 1]
```diff
@@ -1,3 +1,5 @@
+'use strict';
+
```
Comment (ripleyaffect) [inline src/analytics/actions.es6]: Should be added by babel?

**Comment by ripleyaffect** [comment]
👍

File: src/analytics/actions.es6 (hunk 1 above)
Comment (learykara) [inline src/analytics/actions.es6]: was getting an error in stork... something up with the fact that there's no jsx in there?

Comment (learykara) [inline src/analytics/actions.es6]: ah - that error was thrown before I added babel to stork's webpack. makes sense - removing

**Pull request merged**